resources/views/createfaq.blade.php

```diff
@@ -0,0 +1,19 @@
+<x-layout>
+    <x-slot:title>
+        Maak een nieuwe FAQ Post
+    </x-slot>
+    <form method="post" action="createfaqpost">
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" id="question" name="question">
+        <h1>Antwoord: </h1>
+        <input type="text" id="answer" name="answer">
+        <br>
```

Comment:
Using `<br>`  tags is not good practice for layout management. Try to experiment with CSS margins or paddings.